R/collate_results.R

```diff
@@ -4,7 +4,8 @@
 #'
 #' @details This function takes a data frame with sample IDs (in the first column) with the `sample_table` parameter and adds sample-level results from many of the available GAMBL pipelines.
 #' Optional parameters are `these_samples_metadata` and `join_with_full_metadata`. If `join_with_full_metadata` is set to TRUE, the function can either work with an already subset metadata
-#' table (`these_sampels_metadata`), or, if not provided, the function will default to all metadata returned with `get_gambl_metadata`, allowing the user to extend the available information in a metadata table.
+#' table (`these_samples_metadata`), or, if not provided, the function will default to all metadata returned with `get_gambl_metadata`, allowing the user to extend the available information in a metadata table. 
+#' If both `sample_table` and `these_samples_metadata` are set, the function will prioritize and exclusively use the samples in `these_samples_metadata`.
```

Comment:
What is the purpose of sample_table and when/why would it ever be used instead of these_samples_metadata? If it requires sample_id and seq_type isn't it basically redundant?